packages/tanstack-solid-query/test/useSubscription.test.tsx

```
@@ -0,0 +1,465 @@
+    // ignore
+  };
+  console.log = noop;
+  console.error = noop;
+
+  function cleanup() {
+    console.log = log;
+    console.error = error;
+  }
+
+  return makeResource(cleanup, cleanup);
+};
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)